client/modules/IDE/components/Preferences/index.jsx

```diff
@@ -117,7 +117,7 @@ class Preferences extends React.Component {
                   value="light"
```

Comment:
The `aria-label`s need to be translated in these files too as they're used by screen readers.